emails/application_approve/html.pug

```diff
@@ -11,7 +11,7 @@ html
                             tbody
                                 tr
                                     td(align='center', bgcolor='#343A40', style='padding: 40px 0 30px 0; color: #ffffff; font-size: 28px; font-weight: bold; font-family: Arial, sans-serif;')
-                                        img(src='https://cubecobra.com/content/banner.png', alt='Cube Cobra', height='50', style='display: block;')
+                                        img(src=`${baseUrl}/content/banner.png`, alt='Cube Cobra', height='50', style='display: block;')
```

Comment:
FYI I found out that by default the email-template library will inline, as base64 data, images smaller than a default threshold (because we have juiceResources.webResources.image = true, per https://github.com/forwardemail/email-templates?tab=readme-ov-file#options). However many email clients including gmail don't support base64 data per https://www.caniemail.com/features/image-base64/, causing the image not to show.

So wondering if we want to disable that option (or ignore with data-inline-ignore attributes) so the image loads via the web?